src/commands/init/ai-rules.ts

```diff
@@ -0,0 +1,447 @@
+    const errorMessage = error instanceof Error ? error.message : 'Unknown error'
+    log(`${chalk.yellow('⚠️')} Warning: Failed to save AI instructions: ${errorMessage}`)
+  }
+}
+
+const cloneRepo = async (repoUrl: string, targetDir: string, debug: boolean): Promise<void> => {
+  try {
+    await runGit(['clone', repoUrl, targetDir], !debug)
+  } catch (error) {
+    throw new Error(`Failed to clone repository: ${error instanceof Error ? error.message : error?.toString() ?? ''}`)
+  }
+}
+
+/**
+ * Handles AI rules initialization workflow
+ * This is the experimental --ai-rules functionality for the init command
+ */
+export const initWithAiRules = async (hash: string, command: BaseCommand): Promise<void> => {
+  // Authenticate first before any API operations
+  await command.authenticate()
+  const { api } = command.netlify
+
+  log(`${chalk.blue('🤖 Initializing AI project')} with rules...`)
+  log(`${chalk.gray('Hash:')} ${hash}`)
+  log(`${chalk.gray('User:')} ${api.accessToken ? 'Authenticated ✅' : 'Not authenticated ❌'}`)
+
+  try {
+    // Step 1: Decode hash and fetch project information
+    log('\n📋 Decoding project hash...')
+    const decodedUrl = decodeHash(hash)
+    log(`${chalk.cyan('Decoded URL:')} ${decodedUrl}`)
+
+    log('\n🔍 Fetching project information...')
+    const projectInfo = await fetchProjectInfo(decodedUrl)
+
+    log(`${chalk.cyan('Project ID:')} ${projectInfo.projectId}`)
+
+    // Step 2: Get repository URL from project ID via Netlify site API
+    log('\n🔗 Linking to Netlify site and fetching repository...')
+    const repositoryUrl = await getRepoUrlFromProjectId(api, projectInfo.projectId)
+    log(`${chalk.cyan('Repository:')} ${repositoryUrl}`)
+
+    // Step 3: Clone repository
+    const { repoUrl, repoName } = normalizeRepoUrl(repositoryUrl)
+    const targetDir = `ai-project-${repoName}-${hash.substring(0, 8)}`
+
+    const cloneSpinner = startSpinner({ text: `Cloning repository to ${chalk.cyan(targetDir)}` })
+
+    await cloneRepo(repoUrl, targetDir, false)
+    cloneSpinner.success({ text: `Cloned repository to ${chalk.cyan(targetDir)}` })
+
+    // Step 4: Save AI instructions to file
+    if (projectInfo.prompt) {
+      const ntlContext = await fetch(
+        'https://docs.netlify.com/ai-context/scoped-context?scopes=serverless,blobs,forms',
+        {
+          method: 'GET',
+          headers: {
+            'Content-Type': 'text/plain',
+          },
+        },
+      )
+        .then((res) => res.text())
+        .catch(() => {
+          return null
+        })
+      log('\n📝 Saving AI instructions...')
+      await savePrompt(projectInfo.prompt, ntlContext, targetDir)
+    }
+
+    // Step 5: Detect IDE and configure MCP server
+    log('\n🔍 Detecting development environment...')
+    const detectedIDE = await detectIDE()
+    let mcpConfigured = false
+
+    if (detectedIDE) {
+      log(`${chalk.green('✅')} Detected IDE: ${chalk.cyan(detectedIDE.presentedName)}`)
```

Comment:
Can we settle on either "development environment" or "IDE"? Using both next to each other feels unnecessarily confusing.